app/controllers/applications/supplementals_controller.rb

```diff
@@ -8,5 +8,8 @@ class SupplementalsController < ApplicationController
     def update
       application_service.update
     end
+
+    def show
```

Comment:
wondering why this needs to be here or what it's doing? Maybe a comment could help if it's intentional?

Reply:
you're right, this shouldn't be here. I thought I needed it to fix the routes but that was a separate issue